### PR TITLE
Add "amount" field to "wallet inscriptions" output.

### PR DESCRIPTION
**Pull request by gmart7t2**
Allow people to check the postage on each of their inscriptions.

### CODE REVIEW

**Comment by rot13maxi** [comment]
this is always going to be 10000 until we have an option to change the output postage amount, or am I missing something?

**Comment by raphjaph** [comment]
When sending around inscriptions the postage will decrease over time. It will only be replenished if it would go below the dust limit. So this change makes sense.

**Review by raphjaph — CHANGES_REQUESTED** [review]
Reviewing with Casey. This is missing a few things that he will address in his review.

File: src/subcommand/wallet/inscriptions.rs [hunk 1]
```diff
@@ -5,6 +5,7 @@ pub struct Output {
   pub inscription: InscriptionId,
   pub location: SatPoint,
   pub explorer: String,
+  pub amount: u64,
```
Comment (casey) [inline src/subcommand/wallet/inscriptions.rs:8]: I think `postage` is clearer than `amount` here:

```suggestion
  pub postage: u64,
```

Comment (casey) [inline src/subcommand/wallet/inscriptions.rs:8]: Struct fields should be sorted.

File: src/subcommand/wallet/inscriptions.rs [hunk 2]
```diff
@@ -29,6 +30,7 @@ pub(crate) fn run(options: Options) -> Result {
         location,
         inscription,
         explorer: format!("{explorer}{inscription}"),
+        amount: unspent_outputs.get(&location.outpoint).unwrap().to_sat(),
```
Comment (casey) [inline src/subcommand/wallet/inscriptions.rs:33]: Refactor to use `if let …` and avoid the unwrap, like so:

```
    if let Some(output) = unspent_outputs.get(&location.outpoint) {
      output.push(Output {
        location,
        inscription,
        explorer: format!("{explorer}{inscription}"),
        amount: output.to_sat(),
```

**Review by casey — CHANGES_REQUESTED** [review]
Nice! This is a good addition.

This requires a test! We should have an integration test that the returned `postage` field is correct. Add one in `tests/wallet/inscriptions.rs`.

**Review by raphjaph — APPROVED** [review]

**Pull request merged**